### PR TITLE
Addbackground image and responsive

### LINKED ISSUE #112: As a developer I need to fix the bug in Signup Page and add image to Landing page

### PR DESCRIPTION
**Pull request by nuriyealp**
## Description

I added an image to the landing page and I made the page responsive for tablet.

## Resouce links or research articles

https://stackoverflow.com/questions/60682426/material-ui-adds-padding-to-the-body-tag-when-a-dialog-is-opened/74577365#74577365
## Related Issue
Closes #112 

## Acceptance Criteria
Add an image to the landing page
Make the page responsive to tablet

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### After

Screen Shot 2022-11-25 at 7 35 37 PM

### CODE REVIEW

**Review by elenamagay — CHANGES_REQUESTED** [review]
The background looks good but when I switch to tablet or narrower screen there is a big gap between the description and Sign in. Also, please resolve error in the console log. Thanks

**Comment by nuriyealp** [comment]
@elenamagay Thank you.
I made the gap smaller and resolved the error in the console log.

**Review by XeniyaDob — CHANGES_REQUESTED** [review]
Browser screen:
The description and sign-in form are too wide, in my opinion.
Screen Shot 2022-11-29 at 9 45 24 AM

Could you keep this width and center the content?
Screen Shot 2022-11-29 at 9 46 48 AM

Ipad and mobile screens:  The content of the page should be centered. There is no margin on the top. 
Screen Shot 2022-11-29 at 9 44 53 AM

**Comment by nuriyealp** [comment]
I added the margin top of the page. The content of the page is centered now.
I adjusted the picture. The description and sign-in form are not too wide on the Browser screen now.

**Review by XeniyaDob — APPROVED** [review]
Great job, thank you!

File: src/components/Layout/LayoutPublic.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Outlet } from "react-router-dom";
 import Container from "@mui/material/Container";
 
 import FooterPublic from "../Footer/FooterPublic";
-const LayoutPublic = ({ children }) => {
+const LayoutPublic = ({ children, theme }) => {
```
Comment (elenamagay) [inline src/components/Layout/LayoutPublic.js:7]: why did you declare theme here if you never use it in this file?

**Review by elenamagay — APPROVED** [review]

**Review by elenamagay — APPROVED** [review]

**Pull request merged**